package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "coverage": "./node_modules/.bin/nyc --reporter=lcov --reporter=text --reporter=json-summary npm test",
+    "docs": "./node_modules/.bin/jsdoc2md --template ./docs/template.hbs --files ./src/*.js > ./docs/docs.md",
```

Comment:
This task would be handy to call as `prepublishOnly`, see https://docs.npmjs.com/misc/scripts

Reply:
Fixed in 31692eb